crates/pixi_core/src/lock_file/install_subset.rs

```diff
@@ -0,0 +1,458 @@
+use itertools::Itertools;
```

Comment:
This the file that contains the graph reachability stuff and is kind of the meat of this feature.